gi_loadouts/face/scan/rule.py

```diff
@@ -6,26 +6,26 @@
 from PySide6.QtGui import QDragEnterEvent, QDropEvent, QPixmap
 from PySide6.QtWidgets import QApplication, QComboBox, QDialog, QLineEdit, QMessageBox
 
-from gi_loadouts import conf
-from gi_loadouts.data.arti import ArtiList
-from gi_loadouts.face.scan import areaiden, tab_order_scan
-from gi_loadouts.face.scan.file import file
-from gi_loadouts.face.scan.scan import Ui_scan
-from gi_loadouts.face.scan.work import ScanWorker
-from gi_loadouts.face.util import modify_datatype_to_transfer, truncate_text
-from gi_loadouts.face.wind.talk import Dialog
-from gi_loadouts.type import arti
-from gi_loadouts.type.arti import ArtiLevl, __artistat__
-from gi_loadouts.type.arti.base import (
+from ... import conf
+from ...data.arti import ArtiList
+from ...face.scan import areaiden, tab_order_scan
+from ...face.scan.file import file
+from ...face.scan.scan import Ui_scan
+from ...face.scan.work import ScanWorker
+from ...face.util import modify_datatype_to_transfer, truncate_text
+from ...face.wind.talk import Dialog
+from ...type import arti
+from ...type.arti import ArtiLevl, __artistat__
+from ...type.arti.base import (
```

Comment:
Are these many dots needed?

```suggestion
from .. import conf
from ..data.arti import ArtiList
from ..face.scan import areaiden, tab_order_scan
from ..face.scan.file import file
from ..face.scan.scan import Ui_scan
from ..face.scan.work import ScanWorker
from ..face.util import modify_datatype_to_transfer, truncate_text
from ..face.wind.talk import Dialog
from ..type import arti
from ..type.arti import ArtiLevl, __artistat__
from ..type.arti.base import (
```